source/ios/Additions.cs

```diff
@@ -72,14 +72,17 @@ static void ConvertToNsExceptionAndAbort(object e)
 			var msg = e.ToString();
 
 			var ex = e as Exception;
-			if(ex != null) 
-				name = string.Format("{0}: {1}", ex.GetType().FullName, ex.Message);
-
+			if (ex != null) {
+				name = ex.GetType ().FullName;
+				if (ex.StackTrace != null) {
+					msg = msg.Insert (msg.IndexOf('\n'), "Xamarin Exception Stack:");
```

Comment:
Should this not be

`msg = msg.Insert (msg.IndexOf('\n') + 1, "Xamarin Exception Stack:");`

This is inserting before the new line and creating this stack trace:

```
*** Terminating app due to uncaught exception 'TestHockeyApp.TestHockeyAppException', reason: 'TestHockeyApp.TestHockeyAppException: This is a basic exception.Xamarin Exception Stack:
  at TestHockeyApp.TestPage.ThrowBasicException (System.Object sender, System.EventArgs e) [0x00006] in /Users/michaeldang/Projects/TestHockeyApp/TestHockeyApp/TestPage.cs:83 
  at Xamarin.Forms.Button.Xamarin.Forms.IButtonController.SendClicked () [0x00020] in :0 
  at Xamarin.Forms.Platform.iOS.ButtonRenderer.OnButtonTouchUpInside (System.Object sender, System.EventArgs eventArgs) [0x0000e] in :0 
  at UIKit.UIControlEventProxy.Activated () [0x00007] in /Users/builder/data/lanes/3051/5f11db87/source/maccore/src/UIKit/UIControl.cs:37 
  at (wrapper managed-to-native) UIKit.UIApplication:UIApplicationMain (int,string[],intptr,intptr)
  at UIKit.UIApplication.Main (System.String[] args, IntPtr principal, IntPtr delegate) [0x00005] in /Users/builder/data/lanes/3051/5f11db87/source/maccore/src/UIKit/UIApplication.cs:77 
  at UIKit.UIApplication.Main (System.String[] args, System.String principalClassName, System.String delegateClassName) [0x00038] in /Users/builder/data/lanes/3051/5f11db87/source/maccore/src/UIKit/UIApplication.cs:61 
  at TestHockeyApp.iOS.Application.Main (System.String[] args) [0x00008] in /Users/michaeldang/Projects/TestHockeyApp/iOS/Main.cs:17 '
```

https://rink.hockeyapp.net/manage/apps/307767/app_versions/1/crash_reasons/118175066